### PR TITLE
no more require BroadCastChannel API for calendar event updates

### PR DESCRIPTION
**Pull request by tofixx**
changed messages to use postMessage interface instead

### CODE REVIEW

File: static/scripts/calendar.js [hunk 1]
```diff
@@ -55,20 +55,22 @@ $(document).ready(function () {
         editable: false,
         timezone: 'UTC',
         events: function (start, end, timezone, callback) {
+            if('serviceWorker' in navigator){
+                navigator.serviceWorker.addEventListener('message', function (event) {
+                    if (event.origin !== 'https://schul-cloud.org' || 'http://localhost:3100')
```
Comment (janrenz) [inline static/scripts/calendar.js]: this should come from the cofig, as we run on other domains as well

Comment (tofixx) [reply static/scripts/calendar.js]: maybe we just can remove that check. this is a litte security check but the check for data.tag should be enough as well

Comment (janrenz) [reply static/scripts/calendar.js]: i am ok with removing

Comment (tofixx) [reply static/scripts/calendar.js]: solved: we can auto detect this

**Comment by kremer-io** [comment]
@janrenz @tofixx pls always use squash and merge

**Pull request merged**